index.js

```diff
@@ -101,3 +101,11 @@ module.exports.reset = function () {
 	// resets google setup
 	google.reset();
 };
+
+module.exports.refreshGoogleToken = function (cb) {
+	
+	google.refreshToken(function(error, response){
```

Comment:
No need for an extra anonymous function here b/c we are calling cb(error, response);